modules/programs/wezterm.nix

```
@@ -44,13 +45,45 @@ in {
         how to configure.
       '';
     };
+
+    colorschemes = mkOption {
+      type = types.attrsOf (tomlFormat.type);
+      default = { };
+      example = literalExpression ''
+        colorschemeName = {
```

Maybe something like
```suggestion
        myCoolTheme = {
```
or something would be clearer?